lib/structures/Message.js

```diff
@@ -83,6 +83,10 @@ class Message extends Base {
             } else {
                 this.member = null;
             }
+
+            if(!this.guildID) {
+                this.guildID = this.channel.guild.id;
```

Comment:
Please make this check whether the guild is actually present on a channel, as `getMessage` can be executed on a DM channel as well.

Something like
```suggestion
                this.guildID = this.channel.guild && this.channel.guild.id;
```
or
```suggestion
                if (this.channel.guild) {
                    this.guildID = this.channel.guild.id;
                }
```
would suffice, in my opinion.

Reply:
This has already been checked in L77, it is all in the same block

Reply:
Oh I'm blind, disregard...